azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -275,7 +278,8 @@ internal class CallingViewModel(
         }
 
     private fun shouldUpdateRemoteParticipantsViewModels(state: ReduxState) =
-        state.callState.callingStatus == CallingStatus.CONNECTED
+        state.callState.callingStatus == CallingStatus.CONNECTED ||
```

Comment:
Can be slightly reduced if using array,

i.e.
[CallingStatus.CONNECTED, CallingStatus.REMOTE_HOLD].contains(state.callState.callingStatus)
Makes the logic a bit more explicit, and easier to add new states too.

Reply:
As per discussion we will fix for GA.